src/commands/Quotes/addQuote.ts

```diff
@@ -8,7 +8,7 @@ import {addQuote} from '../../database/models/QuoteModel';
 
 const {Command} = require('@sapphire/framework');
 
-const EMBED_FIELD_MAX_CHAR_LENGTH = 1024;
+const EMBED_FIELD_MAX_CHAR_LENGTH = 1022;
```

Comment:
Why the lowering?

Reply:
I display the command using back ticks to make it look nicer, so I'm accounting for those two characters.